mdpow/fep.py

```diff
@@ -150,8 +152,16 @@
 from numkit.observables import QuantityWithError
 from glob import glob
 
+# Patch due to slightly different library names
+try:
+    from ConfigParser import NoOptionError
+except ImportError:
+    from configparser import NoOptionError
```

Comment:
Replace with the canonical solution
```suggestion
from six.moves.configparser import NoOptionError
```
Fewer lines is better.